tests/e2e-aio/e2e_test.go

```diff
@@ -123,6 +123,7 @@ var _ = Describe("E2E test", func() {
 					openai.ImageRequest{
 						Prompt: "test",
 						Size:   openai.CreateImageSize512x512,
+						Model:  "stablediffusion", // Specify this to avoid relying on ListModels order?
```

Comment:
not needed, we actually check for a "stablediffusion" model by default if none is specified (only in the image endpoints). In the AIO tests we are just making sure that the client has to do no changes to start creating images